web/client/utils/__tests__/LayersUtils-test.js

```diff
@@ -1000,4 +1000,22 @@ describe('LayersUtils', () => {
                 availableStyles: [{ name: 'generic' }]
             });
     });
+
+    it('test findGeoServerName with array url', () => {
```

Comment:
just a minor thing:
can you move new tests near the one that are testing findGeoServerName func ?
thanks